### ISSUE TITLE
[BUG] CSA's/ Advisors page

### ISSUE THREAD

**Issue by janinakimtrohlev** [buerokratt/Analytics-Module#64]

**Comment by 1AhmedYasser** [comment]
@PaulaMerle Regarding `If you choose metric Conversation time by advisor you can't choose needed CSA (might be caused by missing data in our test environment).`

it wasn't specified in the task as additional options in the metrics
https://github.com/buerokratt/Analytics-Module/issues/10

Have I missed something ?

**Comment by PaulaMerle** [comment]
@1AhmedYasser you are correct! If the chart displays each CSA conversation duration separately, then the task is completed correctly.
